packages/mobile/src/components/scrubber/Slider.tsx

```diff
@@ -142,11 +146,12 @@ export const Slider = memo((props: SliderProps) => {
   useEffect(getRailPageX, [railRef])
 
   const currentAnimation = useRef<Animated.CompositeAnimation>()
+  const playbackRate = useSelector(getPlaybackRate)
   const play = useCallback(
     (timeRemaining: number) => {
       currentAnimation.current = Animated.timing(translationAnim, {
         toValue: railWidth,
-        duration: timeRemaining,
+        duration: timeRemaining / parseFloat(playbackRate),
```

Comment:
i am getting a little worried about the parseFloats, instead of it being a number?

Reply:
I can add a function that reads from a playback rate map as a middle ground so that parseFloat is not used everywhere